site/src/pages/ManagementSettingsPage/SidebarView.tsx

```diff
@@ -167,19 +133,77 @@ function urlForSubpage(organizationName: string, subpage: string = ""): string {
   return `/organizations/${organizationName}/${subpage}`;
 }
 
+interface OrganizationsSettingsNavigationProps {
+  /** The active org name if an org is being viewed. */
+  activeOrganizationName: string | undefined;
+  /** Organizations and their permissions or undefined if still fetching. */
+  organizations: [Organization, AuthorizationResponse][] | undefined;
+  /** Site-wide permissions. */
+  permissions: AuthorizationResponse;
+}
+
+/**
+ * Displays navigation for all organizations and a create organization link.
+ *
+ * If organizations or their permissions are still loading, show a loader.
+ *
+ * If there are no organizations and the user does not have the create org
+ * permission, nothing is displayed.
+ */
+const OrganizationsSettingsNavigation: FC<
+  OrganizationsSettingsNavigationProps
+> = (props) => {
+  // Wait for organizations and their permissions to load in.
+  if (!props.organizations) {
+    return <Loader />;
+  }
+
+  if (
+    props.organizations.length <= 0 &&
+    !props.permissions.createOrganization
+  ) {
+    return null;
+  }
```

Comment:
also we typically destructure `props` in our React components rather than accessing it directly, and I like the consistency. this should look more like...

```suggestion
> = ({ organizations, permissions, whateverElse }) => {
  // Wait for organizations and their permissions to load in.
  if (!organizations) {
    return <Loader />;
  }

  if (
    organizations.length <= 0 &&
    !permissions.createOrganization
  ) {
    return null;
  }
```

Reply:
Awww bummer, I am the opposite.  Will change for consistency though.